### PR TITLE
Updating change log

### LINKED ISSUE #453: Keep A ChangeLog
In order to provide a clearer sense of mission and as a by-product, produce a clear changelog with

a) intuitive intrinsic diffing support
b) good summaries for reviewers like me

I'd suggest using the excellent https://keepachangelog.com/en/1.0.0/ format.

I appreciate that this may seem like busywork at what is a busy time for you; I share this out of remembering how happy I personally was when I finally took my head out of the sand and started keeping one! Examples [1](https://github.com/jet/propulsion/blob/master/CHANGELOG.md) [2](https://github.com/jet/equinox/blob/master/CHANGELOG.md)

Please feel free to just close this if it really Aint Gonna Happen ;)

### PR DESCRIPTION
**Pull request by j82w**
# Pull Request Template

## Description

Updating the change log to use new format. Removed old preview releases since all the changes were tracked. Added all the changes since the release candidate was shipped.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Closing issues

Put closes #453

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,36 @@
-
 # Microsoft Azure Cosmos DB .NET SDK Version 3.0
-This project provides a client tools or utilities in .NET that makes it easy to interact with Azure Cosmos DB. Azure cosmos DB is published with nuget name [Microsoft.Azure.Cosmos](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/). 
 
-## SDK Public Preview announcement
-The .NET SDK 3.0 is currently in [Public Preview](https://azure.microsoft.com/blog/azure-cosmos-dotnet-sdk-version-3-0-now-in-public-preview/).
-   
+This project provides a client tools or utilities in .NET that makes it easy to interact with Azure Cosmos DB. Azure cosmos DB is published with nuget name [Microsoft.Azure.Cosmos](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/).
```
Comment (bartelink) [inline README.md]: This badly needs a rewording.

File: changelog.md [hunk 2]
```diff
@@ -5,10 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [3.0.0] - July 15, 2019
```
Comment (bartelink) [inline changelog.md]: Date should use same format as below

File: README.md (hunk 1 above)
Comment (kirankumarkolli) [inline README.md]: >> Azure cosmos DB is published with nuget name [Microsoft.Azure.Cosmos](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/).
Only this project is published with that name.

File: README.md [hunk 3]
```diff
@@ -1,23 +1,36 @@
...
+
 ## SDK release notes
-https://github.com/Azure/azure-cosmos-dotnet-v3/blob/master/changelog.md
+
+<https://github.com/Azure/azure-cosmos-dotnet-v3/blob/master/changelog.md>
 
 ### SDK API
-https://docs.microsoft.com/dotnet/api/microsoft.azure.cosmos?view=azure-dotnet-preview
+
+<https://docs.microsoft.com/dotnet/api/microsoft.azure.cosmos?view=azure-dotnet-preview>
```
Comment (kirankumarkolli) [inline README.md]: Remove preview and use https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.cosmos?view=azure-dotnet

**Pull request merged**